main_window/packet_parser.py

```diff
@@ -179,18 +197,14 @@ def message(self, byte_stream: BytesIO, header: Header):
         return data
 
     def event(self, byte_stream: BytesIO, header: Header):
-        """
-
-        :param byte_stream:
-        :type byte_stream:
-        :param header:
-        :type header:
-        :return:
-        :rtype:
-        """
         data: Dict = {}
-        # TODO Enumerate event id -> strings, and convert to human readable string here?
-        data[DataEntryIds.EVENT] = byte_stream.read(1)[0]
+        event_bytes = byte_stream.read(2);
+        event_int = self.bytestoint(event_bytes)
+        data_entry_value = EVENT_IDS[event_int]
+        data[DataEntryIds.EVENT] = data_entry_value.value
```

Comment:
The pythonic way of using enums (especially those with auto values) is without the `.value`. We also need that to discern between enum values and float/numeric values when we handle the data.
```suggestion
        data[DataEntryIds.EVENT] = data_entry_value
```